L1Trigger/L1TGlobal/src/GlobalBoard.cc

```diff
@@ -643,6 +645,24 @@ void l1t::GlobalBoard::runGTL(const edm::Event&,
           }
           //                    delete eSumZdcCondition;
 
+        } break;
+        case CondAXOL1TL: {
+          AXOL1TLCondition* axol1tlCondition = new AXOL1TLCondition(itCond->second, this);
+
+          axol1tlCondition->setVerbosity(m_verbosity);
+
+          axol1tlCondition->evaluateConditionStoreResult(iBxInEvent);
+
+          cMapResults[itCond->first] = axol1tlCondition;
+
+          if (m_verbosity && m_isDebugEnabled) {
+            std::ostringstream myCout;
+            axol1tlCondition->print(myCout);
+
+            edm::LogWarning("L1TGlobal") << "axol1tlCondition " << myCout.str();
+          }
+          //delete axol1tlCCondition;
```

Comment:
Where is the corresponding `delete` for this `new`? Who's responsibility is it? Is there anyway this could be offloaded to a smart pointer to reduce the likelihood of memory leaks?

Reply:
every condition is defined in exactly this way, so this is probably a general question I am not qualified to answer

Reply:
@elfontan Do you know where these conditions get deleted?

Reply:
@elfontan Pinging you again on this.